packages/astro/src/content/data-store.ts

```diff
@@ -77,7 +87,59 @@ export class DataStore {
 		return this.#collections;
 	}
 
-	#saveToDiskDebounced = () => {
+	addAssetImport(assetImport: string, filePath: string) {
+		const id = imageSrcToImportId(assetImport, filePath);
```

Comment:
Is there a world where this is "asset kind"-agnostic? We sometimes get requests for supporting things like videos and general related assets

Reply:
Yeah, I think it could, but I think we'd need to work out a way to separate them, as there's quite a bit of Astro Image-specific code in all of the asset handling stuff